lib/Tool/AssetsInstaller.php

```diff
@@ -129,18 +129,32 @@ private function configureOptions(OptionsResolver $resolver)
         $composerJsonSetting = $this->readComposerJsonSetting();
         if (null !== $composerJsonSetting) {
             if ('symlink' === $composerJsonSetting) {
-                $defaults = array_merge([
-                    'symlink' => true,
-                    'relative' => false,
-                ], $defaults);
+                $defaults = array_merge(
+                    $defaults,
+                    [
+                        'symlink' => true,
+                        'relative' => false,
+                    ]
+                );
             } elseif ('relative' === $composerJsonSetting) {
-                $defaults = array_merge([
-                    'symlink' => true,
-                    'relative' => true,
-                ], $defaults);
+                $defaults = array_merge(
+                    $defaults,
+                    [
+                        'symlink' => true,
+                        'relative' => true,
+                    ]
+                );
             }
         }
 
+        if(in_array(getenv('SYMFONY_ASSETS_INSTALL'), ['symlink', 'relative'])) {
```

Comment:
Probably better to use `$_SERVER` here, right? Otherwise `.env` won't work?

Reply:
You are right. So also change https://github.com/pimcore/pimcore/blob/abdca558837f67183845260bc1250ee37936ac52/lib/Composer.php#L192 ?

Reply:
👍 yep

Reply:
Done